src/m365/flow/commands/flow-get.spec.ts

```diff
@@ -12,6 +12,9 @@ import commands from '../commands.js';
": true, "schema": { "$ref": "#/definitions/NotificationDefinition" } }], "responses": { "200": { "description": "OK" }, "400": { "description": "Bad Request" }, "500": { "description": "Internal Server Error" }, "default": { "description": "Operation Failed." } } } }, "/{connectionId}/sendEmailNotification": { "post": { "description": "Sends an email notification to the account you signed in to Microsoft Flow with.", "summary": "Send me an email notification", "operationId": "SendEmailNotification", "x-ms-visibility": "important", "consumes": ["application/json"], "produces": ["application/json"], "parameters": [{ "name": "connectionId", "in": "path", "required": true, "type": "string", "x-ms-visibility": "internal" }, { "name": "NotificationEmailDefinition", "x-ms-summary": "The email notification", "in": "body", "description": "Email notification inputs", "required": true, "schema": { "$ref": "#/definitions/NotificationEmailDefinition" } }], "responses": { "200": { "description": "OK" }, "400": { "description": "Bad Request" }, "500": { "description": "Internal Server Error" }, "default": { "description": "Operation Failed." } } } } } }, "tier": "NotSpecified" } }, "createdTime": "2018-03-23T17:59:35.4407282Z", "lastModifiedTime": "2018-03-23T17:59:37.1164508Z", "templateName": "a04de6ce52984b3db0b907f588994bc8", "environment": { "name": "Default-d87a7535-dd31-4437-bfe1-95340acd55c5", "type": "Microsoft.ProcessSimple/environments", "id": "/providers/Microsoft.ProcessSimple/environments/Default-d87a7535-dd31-4437-bfe1-95340acd55c5" }, "definitionSummary": { "triggers": [{ "type": "Recurrence" }], "actions": [{ "type": "If" }, { "type": "Query" }, { "type": "ApiConnection", "swaggerOperationId": "ListFeedItems", "metadata": { "flowSystemMetadata": { "swaggerOperationId": "ListFeedItems" } }, "api": { "name": "shared_rss", "id": "/providers/Microsoft.PowerApps/apis/shared_rss", "type": "/providers/Microsoft.PowerApps/apis" } }, { "type": "Foreach" }, { "type": "ApiConnection", "swaggerOperationId": "SendEmailNotification", "metadata": { "flowSystemMetadata": { "swaggerOperationId": "SendEmailNotification" } }, "api": { "name": "shared_flowpush", "id": "/providers/Microsoft.PowerApps/apis/shared_flowpush", "type": "/providers/Microsoft.PowerApps/apis" } }, { "type": "Compose" }], "description": "Each day, get an email with a list of all of the top CNN posts from the last day." }, "creator": { "tenantId": "d87a7535-dd31-4437-bfe1-95340acd55c5", "objectId": "da8f7aea-cf43-497f-ad62-c2feae89a194", "userId": "da8f7aea-cf43-497f-ad62-c2feae89a194", "userType": "ActiveDirectory" }, "flowTriggerUri": "https://api.flow.microsoft.com:443/providers/Microsoft.ProcessSimple/environments/Default-d87a7535-dd31-4437-bfe1-95340acd55c5/flows/3989cb59-ce1a-4a5c-bb78-257c5c39381d/triggers/Every_day/run?api-version=2016-11-01", "installationStatus": "Installed", "provisioningMethod": "FromDefinition", "flowFailureAlertSubscribed": true, "referencedResources": [] } };
```

Comment:
I'm missing the new expanded properties in the response. I'd suggest we do add it here, for clarity, even if it's a lot of useless text. For readability, we could include it in a separate `flow-get.mock.ts` file, and then pretty printed.